Packs/Trello/pack_metadata.json

```diff
@@ -0,0 +1,14 @@
+{
+    "name": "Trello",
+    "description": "Trello is a card-based activity tracker.",
```

Comment:
-> Trello is a card-based activity tracker. Utilize this content pack to organize and prioritize your personal and work life using boards, lists, cards, and so on.